lib/pages/chat/chat_context_menu_actions.dart

```diff
@@ -5,7 +5,8 @@ enum ChatContextMenuActions {
   select,
   copyMessage,
   pinMessage,
-  forward;
+  forward,
+  download;
```

Comment:
What can download? file, image, ....
```suggestion
  downloadFile;
```